client/rest-high-level/src/main/java/org/elasticsearch/client/security/support/ApiKey.java

```diff
@@ -98,9 +109,13 @@ public String getRealm() {
         return metadata;
     }
 
+    public Object[] getSortValues() {
```

Comment:
Should we javadoc this method?
We typically don't (and rely on the API docs) but this field is weird, since it only exists for `ApiKey` objects that came from `_query/`

Reply:
Good point. Added, thanks!